core/arch/arm/tee/pta_socket.c

```diff
@@ -0,0 +1,330 @@
+/*
```

Comment:
Good question. There's only test TAs in that directory. This is the real thing.
It should perhaps go into `core/tee` instead as it's not really arch specific.